docs/project/roads-pavement-bridges-prd.md

```
@@ -0,0 +1,465 @@
+    || hex.HasTerrain(MakaMekTerrains.Bridge);
+```
+
+A unit is **on the road** for a given movement step when **both** the source hex and the destination hex satisfy `IsRoadOrPaved()`.
+
+### Movement Cost Substitution
+
+**File:** `src/MakaMek.Map/Models/BattleMap.cs` — segment cost calculation
+
+When the unit is on the road (both hexes paved), the hex entry cost is **1 MP** (road surface cost) regardless of the underlying terrain's `MovementCost`. Elevation change costs are still added on top.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)